src/IceRpc.Templates/Templates/IceRpc-Slice-Client/certs/README.md

```diff
@@ -0,0 +1,4 @@
+# IceRPC SSL test certificates
+
+The QUIC transport version of this template uses the SSL certificate found in this directory. These
```

Comment:
I would not refer to "template" in this README.md.

Users won't see this README when browsing the contents of the NuGet package. They will see it when browsing the file for _their_ project that they happen to create with one of our templates.

Reply:
Good point! I will update the wording.